FileSystemManager/Views/Shared/_Layout.cshtml

```diff
@@ -8,6 +9,14 @@
     <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/FileSystemManager.styles.css" asp-append-version="true" />
 </head>
+<style>
```

Comment:
Why not to add this to site.css in wwwroot?

Reply:
Done